src/opencl/opencl_utils.hpp

```diff
@@ -46,72 +48,93 @@ void getAllPlatforms(cl_platform_id*& platforms, cl_uint& num_platforms, cl_int&
 
 std::string getPlatformName(cl_platform_id& platform_id, cl_int& err) {

```

Comment:
Looks like all calls of this function just throw results away, is it still needed or `getDeviceVendor` is sufficient?